GeoIP2/DatabaseReader.cs

```diff
@@ -39,9 +39,9 @@ public DatabaseReader(string file, FileAccessMode mode = FileAccessMode.MemoryMa
         /// <param name="file">The MaxMind DB file.</param>
         /// <param name="locales">List of locale codes to use in name property from most preferred to least preferred.</param>
         /// <param name="mode">The mode by which to access the DB file.</param>
-        public DatabaseReader(string file, List<string> locales, FileAccessMode mode = FileAccessMode.MemoryMapped)
+        public DatabaseReader(string file, IEnumerable<string> locales, FileAccessMode mode = FileAccessMode.MemoryMapped)
```

Comment:
Is there a test that shows #41 (https://github.com/maxmind/GeoIP2-dotnet/issues/41) is fixed?